server.go

```diff
@@ -233,6 +241,8 @@ type Server struct {
 	// AcceptMsgFunc will check the incoming message and will reject it early in the process.
 	// By default DefaultMsgAcceptFunc will be used.
 	MsgAcceptFunc MsgAcceptFunc
+	// InvalidMsgFunc is optional, will be called if a message is received but cannot be parsed.
+	InvalidMsgFunc InvalidMsgFunc
```

Comment:
MsgInvalidFunc?

to be in sync with `MsgAcceptFunc`

Reply:
Works for me.  Done.